filebeat/filebeat.reference.yml

```diff
@@ -116,6 +116,16 @@ filebeat.modules:
     # Filebeat will choose the paths depending on your OS.
     #var.paths:
 
+#------------------------------- haproxy Module ------------------------------
+- module: haproxy
+  # All logs
+  {fileset}:
```

Comment:
Another placeholder here, remember to run also `make update` after modifying these files.